test/web-platform-tests/run-wpts.js

```diff
@@ -69,7 +70,13 @@ describe("web-platform-tests", () => {
           if (matchingPattern && shouldSkip) {
             specify.skip(`[${reason}] ${testFile}`);
           } else if (expectFail) {
-            runSingleWPT(testFilePath, `[expected fail] ${testFile}`, expectFail);
+            runSingleWPT(
+              testFilePath,
+              `[expected fail${
+                reason !== "fail" ? `: ${reason}` : ""
+              }] ${testFile}`,
```

Comment:
Could you un-inline this back to the code I previously asked you to change it to?